src/Appwrite/Platform/Workers/Builds.php

```diff
@@ -266,7 +273,13 @@ protected function buildDeployment(Device $deviceForFunctions, Func $queueForFun
                 if (!empty($templateRepositoryName) && !empty($templateOwnerName) && !empty($templateBranch)) {
                     // Clone template repo
                     $tmpTemplateDirectory = '/tmp/builds/' . \escapeshellcmd($buildId) . '/template';
-                    $gitCloneCommandForTemplate = $github->generateCloneCommand($templateOwnerName, $templateRepositoryName, $templateBranch, $tmpTemplateDirectory, $templateRootDirectory);
+
+                    $cloneType = GitHub::CLONE_TYPE_BRANCH;
+                    if(\str_starts_with($templateBranch, '0.1.')) { // Temporary fix for 1.5. In future versions this only support tag names
```

Comment:
I am aware of this comment, and will address it in a separate PR that also fixes template for non-vcs functions